README.md

```diff
@@ -145,18 +204,35 @@ Album to look up:  Human
 ┡━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
 │ Human (Deluxe) │ 2               │
 └────────────────┴─────────────────┘
-
 ```
 
-## TODO 📝
+</details>
+
+
+## 🛠️ Library customization
+
+The resulting `Library.csv` file will be automatically updated by Amusing at every DB change.
+
+You can manually modify it to change which YouTube video to download for a specific song.
+You can also add a custom URL to download a specific album artwork.
+
+Here are some great tools you can use to find album artworks:
+- [Ben Dodson's iTunes Artwork Finder](https://bendodson.com/projects/itunes-artwork-finder/)
+- [Ben Dodson's Apple Music Artwork Finder](https://bendodson.com/projects/apple-music-artwork-finder)
+
+Copy the image link into your CSV file and Amusing will download it and embed it into your song!
```

Comment:
Minor: Will Amusing download automatically or will we have to run `amusing download` command again to make it happen? Should we include this additional step here?

Reply:
The songs are automatically downloaded only when running the `download` command. Currently, to edit the values it is necessary to insert them into the `Library.csv` (which has to be generated with a previous `amusing parse` or `amusing download` from the `Library.xml`).

In a certain sense it is necessary to perform the command twice, because, once edited, the file `Library.csv` has to be parsed again.

Reply:
So If you want to specify, we could add that Amusing will download if you parse and download again the modified CSV.

Reply:
Since the `amusing download` will do the two steps, it will be useful to add this here.

Reply:
Something like this?

```suggestion
Copy the image link into your CSV file and Amusing will download it and embed it into your song the next time you run `amusing download '/path/to/Library.csv'`!
```

Reply:
Looks good!

Reply:
I'll leave you the privilege to commit the suggestion then 😉